controllers/comment.controllers.js

```diff
@@ -0,0 +1,23 @@
+const { delComment, fetchCommentById } = require("../models/comment.models");
+
+exports.getCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  fetchCommentById(comment_id)
+    .then((response) => {
+      res.status(200).send({ comment: response });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  Promise.all([fetchCommentById(comment_id), delComment(comment_id)])
+    .then(() => {
+      res.status(204).send();
+    })
```

Comment:
we don't even need the .send() here as we're not sending anything 
a simple status(204) is perfect